tests/mocha/.eslintrc.json

```diff
@@ -14,6 +14,7 @@
         "assertVariableValues": true,
         "captureWarnings": true,
         "createTestBlock": true,
+        "createRenderedBlock": true,
```

Comment:
minor nit: I'd prefer this to be swapped in order with the one above so that its alphabetical